egs/iam/v2/run.sh

```diff
@@ -0,0 +1,200 @@
+#!/bin/bash
```

Comment:
Is there an obvious gain from using this script? If I'm right, the results are almost the same. I guess it's better to keep v2 small and only add scripts which give a clear improvement.

Reply:
Thanks, I am trying to tune run_end2end.sh to give better results. currently, run.sh is giving a slightly better result as compared to run_end2end.sh (9.0 vs 9.3). I will remove it.

Reply:
I see, but the results are the same on the other 3 test conditions so this 0.3 difference could be noise.